seatunnel-api/src/main/java/org/apache/seatunnel/api/table/schema/event/AlterTableModifyColumnEvent.java

```diff
@@ -29,6 +29,7 @@
 public class AlterTableModifyColumnEvent extends AlterTableColumnEvent {
     private final Column column;
     private final boolean first;
+    private boolean typeChanged = false;
```

Comment:
```suggestion
    private Boolean typeChanged;
```


Any default value is incorrect